### DISCUSSION TITLE
Monica - open source personal CRM.

### DISCUSSION THREAD

**Discussion by 391a** [community-scripts/ProxmoxVE#972] [Request script]
### Application Name

Monica

### Website

https://www.monicahq.com/

### Description

Personal CRM. Remember everything about your friends, family and business relationships.
https://github.com/monicahq/monica

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Duplicate. Use search function before
